qa/crontab/teuthology-cronjobs

```diff
@@ -0,0 +1,271 @@
+40 05 * * 2,4,6  CEPH_BRANCH=luminous; MACHINE_NAME=ovh;   /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s rest -k distro -e $CEPH_QA_EMAIL
+
+#two suites below MUST use --suite-branch hammer OR jewel
+# filtering --filter="rbd_cli_import_export" b/c of bug http://tracker.ceph.com/issues/21573 
+45 05 * * *  CEPH_BRANCH=luminous; MACHINE_NAME=ovh; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s upgrade/client-upgrade-hammer -k distro -e $CEPH_QA_EMAIL --suite-branch hammer
+47 05 * * *  CEPH_BRANCH=luminous; MACHINE_NAME=ovh; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s upgrade/client-upgrade-jewel -k distro -e $CEPH_QA_EMAIL --suite-branch jewel
+50 05 * * *  CEPH_BRANCH=luminous; MACHINE_NAME=ovh; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s upgrade/client-upgrade-kraken -k distro -e $CEPH_QA_EMAIL --suite-branch kraken
+# removed ceph-deploy per Vasu reuest 2/2/18
+#55 05 * * 2,4,6  CEPH_BRANCH=luminous; MACHINE_NAME=ovh;    /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s ceph-deploy -k distro ~/vps.yaml -e $CEPH_QA_EMAIL
+10 05 * * 2,4,6  CEPH_BRANCH=luminous; MACHINE_NAME=ovh;    /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s ceph-disk -k distro ~/vps.yaml -e $CEPH_QA_EMAIL
+15 05 * * 2,4,6  CEPH_BRANCH=luminous; MACHINE_NAME=ovh;    /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s ceph-ansible -k distro -e $CEPH_QA_EMAIL
+
+##########################
+
+
+
+
+# disabled pre Sage 05 18 * * 2,4,6   CEPH_BRANCH=infernalis; MACHINE_NAME=vps; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -k distro -m $MACHINE_NAME -s upgrade/hammer-x ~/vps.yaml -e $CEPH_QA_EMAIL
+#NO SUPPORT 10 18 * * 1,7     CEPH_BRANCH=infernalis; MACHINE_NAME=vps; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -k distro -m $MACHINE_NAME -s upgrade/infernalis ~/vps.yaml -e $CEPH_QA_EMAIL
+15 01 * * 1,5    CEPH_BRANCH=jewel; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -k distro -m $MACHINE_NAME -s upgrade/hammer-x ~/vps.yaml -e $CEPH_QA_EMAIL
+#NO SUPPORT 20 17 * * *       CEPH_BRANCH=jewel; MACHINE_NAME=vps; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -k distro -m $MACHINE_NAME -s upgrade/infernalis-x ~/vps.yaml -e $CEPH_QA_EMAIL
+
+###### on vps
+#20 04 * * 7      CEPH_BRANCH=kraken; MACHINE_NAME=vps; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -k distro -n 7 -m $MACHINE_NAME -s upgrade/jewel-x ~/vps.yaml -e $CEPH_QA_EMAIL
```

Comment:
@yuriw what about vps here?

Reply:
@vasukulkarni they commented out, I will start cleaning crontab item after we finish setup